service/refund_test.go

```diff
@@ -38,8 +38,9 @@ func TestUnitCreateRefund(t *testing.T) {
 			GetGovPayRefundSummary(req, id).
 			Return(nil, nil, Error, err)
 
-		refund, status, err := service.CreateRefund(req, id, body)
+		paymentSession, refund, status, err := service.CreateRefund(req, id, body)
 
+		So(paymentSession, ShouldBeNil)
```

Comment:
why is paymentSession always Nil? looking at these tests I would assume that it's always Nil regardless of the test. Please vary your assertions in unit tests for maximum coverage of different scenarios.

Reply:
Whenever errors occur, paymentSession is nil. In the last, positive test the paymentSession is not nil.